src/Microsoft.DotNet.Darc/src/Darc/Options/VirtualMonoRepo/VmrCommandLineOptions.cs

```diff
@@ -26,26 +26,25 @@ public IServiceCollection RegisterServices()
         var tmpPath = Path.GetFullPath(TmpPath ?? Path.GetTempPath());
         string gitHubToken = null;
         string azureDevOpsToken = null;
+        LocalSettings localDarcSettings = null;
 
         try
         {
-            var localDarcSettings = LocalSettings.LoadSettingsFile(this);
-            gitHubToken = GitHubPat ?? localDarcSettings?.GitHubToken;
-            azureDevOpsToken = AzureDevOpsPat ?? localDarcSettings?.AzureDevOpsToken;
+            localDarcSettings = LocalSettings.LoadSettingsFile(this);
```

Comment:
I am thinking - we should only try to load the local settings if both tokens are null so we are not trying to read it always for no reason.

Reply:
what if only one is null and we can read the other one from the settings?

Reply:
Then we should read it from the settings